corehq/apps/userreports/templates/userreports/expression_debugger.html

```diff
@@ -0,0 +1,58 @@
+{% extends "userreports/userreports_base.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% block js %}{{ block.super }}
+    <script src="{% static 'userreports/js/expression_evaluator.js' %}"></script>
+{% endblock %}
+{% block js-inline %}{{ block.super }}
+<script type="text/javascript">
+    $(function () {
+        var ExpressionModel = hqImport('userreports/js/expression_evaluator.js').ExpressionModel;
+        var submitUrl = "{% url 'expression_evaluator' domain %}";
+        var expressionEditor = $('.CodeMirror')[0].CodeMirror;  // http://stackoverflow.com/a/24987585/8207
+        var sampleExpression = {
+            "type": "property_name",
+            "property_name": "name"
+        };
+        var sampleText = JSON.stringify(sampleExpression, null, 2);
+        expressionEditor.setValue(sampleText);
+        ko.applyBindings(new ExpressionModel(expressionEditor, submitUrl), $("form#expression-debugger").get(0));
```

Comment:
nitpicky, but `#expression-debugger` is more performant than `form#expression-debugger`, and `document.getElementById('expression-debugger')` more performant than that, since you don't need the jQuery-ified version of the element